### PR TITLE
Doc: Replay attack issue

### PR DESCRIPTION
**Pull request by freddyli7**
The document update regarding to transaction replay attack issue.

### CODE REVIEW

File: docs/basics/notice.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+order: 7
+-->
+
+# Notice
+
+Important information for Etheremint development.
+
+## Cross-Chain Transaction Replay Attack Prevention
+
+Given the fact that Ethermint uses the same transaction signature pattern and the type of chainIDEpoch for Ethereum compatibility,
```
Comment (noot) [inline docs/basics/notice.md]: maybe reword to say "Given the fact that Ethermint uses the same transaction signature format as Ethereum for compatibility with Ethereum tooling, and Ethermint's  `chainIDEpoch` corresponds to the chain ID on Ethereum, any valid transaction..."

File: docs/basics/notice.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+-->
+
+# Notice
+
+Important information for Etheremint development.
+
+## Cross-Chain Transaction Replay Attack Prevention
+
+Given the fact that Ethermint uses the same transaction signature format as Ethereum for compatibility with Ethereum tooling, and Ethermint's chainIDEpoch corresponds to the chain ID on Ethereum,
+any valid transaction that is able to broadcast to Ethereum is also able to broadcast to Ethermint, and vice versa, which could lead to the Transaction Replay Attack if Ethermint starts the network by using Ethereum reserved chainIDs.
```
Comment (araskachoi) [inline docs/basics/notice.md:12]: "any valid transaction that is able to be broadcasted to Ethereum is also able to be broadcasted to Ethermint"

Comment (araskachoi) [reply docs/basics/notice.md:12]: -> "... could lead to a transaction replay attack..."

-> "... if Ethermint is initiated using a reserved chainID according to EIP155."

File: docs/basics/notice.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+To prevent this issue, simply choose a different chainID other than [chainIDs that are reserved for Ethereum](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids).
```
Comment (araskachoi) [inline docs/basics/notice.md:14]: "To prevent this issue, please choose a chainID that does not conflict with any of these [reserved ChainIDs](link...)"

**Review by araskachoi — CHANGES_REQUESTED** [review]
looks good, please make the changes and will approve!
